cloud-aws-common/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/common/resource/volume/AwsAttachmentResourceBuilder.java

```diff
@@ -128,6 +128,13 @@ private void checkIfEveryVolumesAttachedSuccessfullyEvenIfSomethingFailed(CloudR
         }
     }
 
+    @Override
+    public List<CloudResource> update(AwsContext context, CloudInstance instance, long privateId,
+        AuthenticatedContext auth, Group group, CloudStack cloudStack) throws Exception {
+        // TODO vertical scaling disks
+        return null;
```

Comment:
might be better to throw UnsupportedOperationException()

Reply:
if the return is null then we just skipping it. with this mechanism we can easily do the disk modification in the future

Reply:
is your question answered @lnardai ? can we merge this?